rb/spec/integration/selenium/webdriver/error_spec.rb

```diff
@@ -47,6 +47,13 @@ module WebDriver
       rescue WebDriver::Error::NoSuchElementError => e
         expect(e.backtrace).not_to be_empty
       end
+
+      it 'has backtrace when using a remote server', only: {driver: :remote} do
+        options = Selenium::WebDriver::Options.chrome
+        Selenium::WebDriver.for :remote, url: 'http://localhost:4444/wd/hub', options: options
```

Comment:
Why would this code fail and raise an error?

Reply:
I updated the options line, to use an incorrect binary

The idea is to produce a session disconnection. In my old code, I tested locally by having a remote that would time out, but this would not work here, so that's why I'm using the incorrect binary now. 

The thing I was looking at was that maybe that URL would not work and produce a TCP error instead of a session error, if the server is not running in that port but I did not figure out how to change the port of the server @p0deje

Reply:
This test would run on all browsers and even non-remote ones. It would also leak the driver and not close when the test is finished. Can you please change it to;

1. Have a `only: {driver: :remote}` guard.
2. Use `create_driver!` helper.
3. some assertion for the backtrace itself (that's what the test is checking based on its title).

Reply:
Thank you for the explanation @p0deje I look more into it, and tried the test locally too and is working, ready for re-review